example-pipelines/environments/dev/terraform.tfvars

```diff
@@ -0,0 +1,5 @@
+project_id = "PROJECT_ID"
+environment = "dev"
+subnet1_region = "us-central1"
+subnet1_zone = "us-central1-a"
+subnet1_cidr = "100.0.0.0/17"
```

Comment:
I think you meant
```suggestion
subnet1_cidr = "10.0.0.0/17"
```

Reply:
prod is 10.x and dev is 100.x to keep the CIDR blocks different since we are using the same project here, although it won't raise errors but not a good practice to keep two VPCs with same CIDR block.

Reply:
@r4hulgupta 100.x threw an error for me during the run
`Error creating Subnetwork: googleapi: Error 400: Invalid value for field 'resource.ipCidrRange': '100.0.0.0/17'. IpCidrRange must specify a range contained in RFC 1918 spaces., invalid[0m`

Reply:
fixed